api/envoy/type/matcher/v3/regex.proto

```diff
@@ -59,7 +59,7 @@ message RegexMatcher {
     // Google's RE2 regex engine.
     GoogleRE2 google_re2 = 1 [
       deprecated = true,
-      (validate.rules).message = {required: true},
+      (validate.rules).message = {required: false},
```

Comment:
Why can't we just remove the validation rule?

Reply:
i thnk you can safely remove - i think this is just setting the default curently

Reply:
Done ✅